scripts/monitoring/auto_scale_workers.py

```diff
@@ -2,22 +2,27 @@
 import time
 import pytz
 import requests
-
-import boto3
-
-import botocore
+import warnings
 
 from auto_stop_workers import start_worker, stop_worker
+from prometheus_api_client import PrometheusConnect
+
+warnings.filterwarnings("ignore")
 
 utc = pytz.UTC
 
+NUM_PROCESSED_SUBMISSIONS = "num_processed_submissions"
+NUM_SUBMISSIONS_IN_QUEUE = "num_submissions_in_queue"
+PROMETHEUS_URL = "https://monitoring-staging.eval.ai/prometheus/"
```

Comment:
@gchhablani can we fetch the endpoint from a environment variable named `MONITORING_API_URL`.